model/team.py

```diff
@@ -0,0 +1,36 @@
+# data model for a slack team
+class Team:
+    __display_name = ""
+    __platform = ""
+    __members = set()
+    __github_team_name = ""
+
+    def __init__(self, display_name):
```

Comment:
One thing I just noticed here: it's probably better to have a unique ID (maybe a uuid? Or just reuse the Github team name?) for the team, so that we can change the display name without changing the team's identity.

Reply:
Ooh ok I'll do that too then thanks!